src/client/lobby/room-list.js

```diff
@@ -22,3 +24,12 @@ export default function RoomList(props) {
         </div>
     );
 }
+
+function joinRoom(arg) {
+    cloak.message('joinRoom', arg.id);
+    redirectToRoom(arg.name);
```

Comment:
This is good, because it reads nicer than browserHistory.push - however, you could use a RoutingService? You could create a top level services folder and then add routing-service.js:
```
import { browserHistory } from 'react-router;

export default {
    navigateTo
};

function navigateTo(path) {
    browserHistory.push(path);
}
```
Then instead of `redirectToRoom(arg.name);` you can have ``RoutingService.navigateTo(`/room/${arg.name}`);``